CHANGELOG

```diff
@@ -1,3 +1,6 @@
+v 25.0.2snap1
```

Comment:
Please remove this, we generate the changelog when creating a release as opposed to on the fly.

Reply:
Done.